recipe/meta.yaml

```diff
@@ -15,11 +15,15 @@ build:
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
+  build:
+    - setuptools >=50
+    - setuptools_scm >=6
+    - wheel
   host:
     - pip
     - python
```

Comment:
Could you please add `toml` to `host`? See https://github.com/nexB/license-expression/blob/v30.0.0/pyproject.toml
`setuptools_scm[toml] >= 6` means both `setuptools_scm >= 6` **AND** `toml`

Reply:
I was not aware of that - thank you!